server/src/resolvers/args/MemberArgs.ts

```diff
@@ -0,0 +1,34 @@
+import { IsEmail, Matches, MinLength } from 'class-validator';
+import { ArgsType, Field } from 'type-graphql';
+
+const passwordRegExp = new RegExp('^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#$%^&*])(?=.{8,})');
```

Comment:
I think regex like this one, or any stuff that can be considered as a real constant should be place in a dedicated file. (e.g: /utils/validation.ts)

Reply:
Good point, i will do this.

Reply:
Created enums/Validations.TS a enum to manage all regex validations